test/test.js

```diff
@@ -51,18 +53,19 @@ describe('request-progress', function () {
         });
 
         request.emit('request');
-        request.emit('response', { headers: { 'content-length': 10 } });
+        response.headers = { 'content-length': 10 };
+        request.emit('response', response);
```

Comment:
Current support is defined here: https://github.com/IndigoUnited/node-request-progress/blob/master/.travis.yml

Though, since I will bump major, we can remove `0.12` which means you can use `Object.assign`. In fact, we could make all code using ES6 but I guess this is out of the scope of this PR.